packages/create-cloudflare/e2e-tests/helpers.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from "fs";
 import crypto from "node:crypto";
 import { tmpdir } from "os";
-import { basename, join } from "path";
+import { basename, join, resolve } from "path";
```

Comment:
usage of these methods would be clearer with the namespace eg `path.resolve` vs `resolve`

and you wouldn't need to abbreviate the Promise resolve callback param variable names below to avoid shadowing